data/remnant ships.txt

```diff
@@ -366,7 +366,7 @@ ship "Pelican"
 		"heat dissipation" 0.7
 		"fuel capacity" 600
 		"ramscoop" 2
-		"outfit space" 470
+		"outfit space" 480
```

Comment:
You just increased the Pelican outfit space substantially and you need to increase it again?

Reply:
Other changes don't really have much bearing on tweaking ships. It was pointed out that having scram drives as default instead of hyperdrives would fit the lore and utilization scenario for the ship. 

More to the point, does the change in space create a potential imbalance (allow for too many bunks re: capture potential, or allow it to become too good of a warship?), or does it simply make it a more interesting variation on the ship type?